softvis3d.properties

```diff
@@ -1,14 +1,20 @@
 category=Visualization/Reporting
 description=Provides 3D models
 homepageUrl=http://softvis3d.com
-archivedVersions=0.5.2,0.6.0,0.7.0,0.8.0,0.8.1,0.9.0,1.0,1.0.1
-publicVersions=1.1.0
+archivedVersions=0.5.2,0.6.0,0.7.0,0.8.0,0.8.1,0.9.0,1.0,1.0.1,1.1.0
+publicVersions=1.2.0
 
 defaults.mavenGroupId=de.rinderle.softvis3d
 defaults.mavenArtifactId=softvis3d
 
+1.2.0.description=New color and filter features.
+1.2.0.sqVersions=[7.6,8.7.*]
+1.2.0.date=2021-04-13
+1.2.0.changelogUrl=https://github.com/stefanrinderle/softvis3d/releases/tag/softvis3d-1.2.0
+1.2.0.downloadUrl=https://github.com/stefanrinderle/softvis3d/releases/download/softvis3d-1.2.0/sonar-softvis3d-plugin-1.2.0.jar
+
 1.1.0.description=Plugin is now adapted to the new directory layout concept.
-1.1.0.sqVersions=[7.6,LATEST]
+1.1.0.sqVersions=[7.6,8.7.*]
```

Comment:
This range needs to end with 8.8

Reply:
Unfortunately, the version 1.1.0 and 1.2.0 do not work with SQ 8.8 yet. I will provide a new release within the next days/weeks for that. But currently, it is not compatible to 8.8 or LATEST.